articles/virtual-machines/windows-in-place-upgrade.md

```diff
@@ -175,7 +175,14 @@ To initiate the in-place upgrade the VM must be in the `Running` state. Once the
    .\setup.exe /auto upgrade /dynamicupdate disable 
    ```
 
-1. Select the correct "Upgrade to" image based on the current version and configuration of the VM using the [Windows Server upgrade matrix](/windows-server/get-started/upgrade-overview).
+   You can use /eula accept switch in the Windows Server upgrade command, to automatically accept the Microsoft Software License Terms (End User License Agreement,     or EULA) during the upgrade process.
+   Using the /eula accept switch can help avoid issues where the upgrade process stalls because the EULA was not accepted manually. This switch ensures that the        upgrade process can proceed smoothly without requiring user interaction to accept the license terms.
```

Comment:
```suggestion
   Using the /eula accept switch can help avoid issues where the upgrade process stalls because the EULA was not accepted manually. This switch ensures that the upgrade process can proceed smoothly without requiring user interaction to accept the license terms.
```